### DISCUSSION TITLE
Can you make it support pojavlauncher pls ?

### DISCUSSION THREAD

**Discussion by Saholi** [xCollateral/VulkanMod#294] [General]
If you don't know sodium doesn't support pojav launcher I mean it literally doesn't increase fps at all and optifine is just bad as vanilla. So if it possible to make this mod support pojav launcher can you guys pls make one we will appreciate it alot. Thank you

**Comment by Felix14-v2** [comment]
Actually there was a fork of this mod for PojavLauncher specially
https://github.com/PojavLauncherTeam/VulkanMod
Requires this launcher branch
https://github.com/PojavLauncherTeam/PojavLauncher/tree/v3_vulkanmod

**Reply by Saholi** [reply]
Oh i didn't know that thanks for the info let me check that
